server/controllers/documentController.js

```diff
@@ -1,132 +1,298 @@
 import db from '../models';
 
 const Documents = db.documents;
+const Users = db.users;
 
+/**
+ * Controller for document management
+ */
 class DocumentController {
-    /**
-     * Method to set the various document routes
-     * @param{Object} app - Express app
-     * @return{Void}
-     */
-    static postRequest(request) {
-        return (
-            request.body &&
-            request.body.title &&
-            request.body.content &&
-            request.body.access &&
-            request.body.OwnerId
-        )
+  /**
+   * Method to set the various document routes
+   * @param{Object} request - Server request
+   * @return{Void} return Void
```

Comment:
I'm not exactly sure how the postRequest method returns void, could explain please?